packages/documentation/docs/controls/charts/overview.md

```diff
@@ -43,9 +76,28 @@ noMargin
 
 ## Empty state
 
+An empty state occurs when a user first opens an application, and no chart was added manually. Make sure to provide a clear message and a call to action to guide users on what to do next.
+
 <Playground
 height="40rem"
 name="echarts-empty-state"
 noMargin
 examplesByName>
 </Playground>
+
+### Loading indicator
+
+A loading indicator provides users with visual feedback that the chart is being processed and will be displayed shortly. This helps manage user expectations and enhances the overall user experience by preventing confusion or frustration during data loading times. 
+
+### Failure and error message
+
+A failure occurs when no data can be displayed within the chart. This can happen for various reasons, such as connection failure, missing data or other. Error messages have following three elements to move users forward:
+
+- State problem: What happened?​ Add a clear reason for the error, i.e. “No data available”
+- Explain cause: Why did the error appear?​ A clear and concise message explaining why the error happened. "Connection failure"
+- Give solution: What can the user do to proceed?​  
+    Add clear instructions for the user regarding what to do next to resolve the error, e.g. “Try again”
+
+### Missing data points
+
+Indicate missing data with a special visual marker (like a different color or shape) to highlight the gaps without connecting them.
```

Comment:
I would like to **see** something here, let it be a coded or figma visualization

Reply:
A code example will be added later